src/jl_uv.c

```diff
@@ -678,56 +678,71 @@ JL_DLLEXPORT int jl_printf(uv_stream_t *s, const char *format, ...)
     return c;
 }
 
-STATIC_INLINE void print_error_msg_as_json(char *buf) JL_NOTSAFEPOINT
+STATIC_INLINE int copystp(char *dest, const char *src)
 {
-    // Our telemetry on SPCS expects a JSON object per line
-    // The following lines prepare the timestamp string and the JSON object
+    char *d = stpcpy(dest, src);
+    return (int)(d - dest);
+}
+
+// RAI-specific
+STATIC_INLINE void write_to_safe_crash_log(char *buf) JL_NOTSAFEPOINT
+{
+    // Our telemetry on SPCS expects a JSON object per line.
+    // We ignore write failures because there is nothing we can do.
+    int buflen = strlen(buf);
+    assert(buflen <= 4096);
```

Comment:
Would it be better to print as much we can and then stop, rather than asserting, and therefore printing nothing?

Reply:
Followed this suggestion.